packages/firebase_ui_localizations/test/firebase_ui_localizations_test.dart

```diff
@@ -0,0 +1,171 @@
+        () async {
+          final localizations = await delegate.load(localeTW);
+          expect(localizations.labels.signInWithPhoneButtonText, '使用電話號碼登入');
+        },
+      );
+
+      testWidgets(
+        'UI test for the "${localeTW.toLanguageTag()}" translation',
+        (tester) async {
+          await tester.pumpWidget(
+            TestMaterialApp(
+              locale: localeTW,
+              localizedText: localizedText,
+            ),
+          );
+          expect(find.text('使用電話號碼登入'), findsOneWidget);
+        },
+      );
+    },
+  );
+
+  group(
+    'Localization override',
+    () {
+      localizedText(BuildContext context) => FirebaseUILocalizations.labelsOf(context).verifyEmailTitle;
+
+      test(
+        'Overrides the DefaultLocalizations',
+        () async {
+          final localizations = await const FirebaseUILocalizationDelegate(
+            DefaultLocalizationsOverrides(),
+          ).load(localeTW);
+          expect(localizations.labels.verifyEmailTitle, 'Overwritten');
+        },
+      );
+
+      testWidgets(
+        'UI test for the default translation override',
+        (tester) async {
+          await tester.pumpWidget(
+            TestMaterialApp(
+              locale: localeZh,
+              localizationsOverride: const FirebaseUILocalizationDelegate(
+                DefaultLocalizationsOverrides(),
+              ),
+              localizedText: localizedText,
+            ),
+          );
+          expect(find.text('Overwritten'), findsOneWidget);
+        },
+      );
+
+      test(
+        'Overrides the DefaultLocalizations',
+        () async {
+          final localizations = await const FirebaseUILocalizationDelegate(
+            ZhTWLocalizationsOverrides(),
+          ).load(localeTW);
+          expect(localizations.labels.verifyEmailTitle, '覆寫標題');
+        },
+      );
+
+      testWidgets(
+        'UI test for the "${localeTW.toLanguageTag()}" translation override',
+        (tester) async {
+          await tester.pumpWidget(
+            TestMaterialApp(
+              locale: localeTW,
+              localizationsOverride: const FirebaseUILocalizationDelegate(
+                ZhTWLocalizationsOverrides(),
+              ),
+              localizedText: localizedText,
+            ),
+          );
+          expect(find.text('覆寫標題'), findsOneWidget);
+        },
+      );
+    },
+  );
+}
+
+class DefaultLocalizationsOverrides extends DefaultLocalizations {
+  const DefaultLocalizationsOverrides();
+
+  @override
+  String get verifyEmailTitle => 'Overwritten';
+}
+
+class ZhTWLocalizationsOverrides extends ZhTWLocalizations {
+  const ZhTWLocalizationsOverrides();
+
+  @override
+  String get verifyEmailTitle => '覆寫標題';
+}
+
+class TestMaterialApp extends StatelessWidget {
+  final Locale locale;
+  final LocalizationsDelegate? localizationsOverride;
+  final String Function(BuildContext context) localizedText;
```

Comment:
`localizedText` sounds more like a data property, not a function, maybe `localizeText` is a better name